src/components/common/ContentCard/ContentCard.scss

```diff
@@ -0,0 +1,30 @@
+.content-card-container {
+    &:hover {
+        box-shadow: 0 8px 12px 0 rgba(30, 35, 96, 0.1), 0 1px 4px 0 rgba(0, 0, 0, 0.1);
+        border-radius: 400;
+    }
+
+    .Horizontal {
```

Comment:
Why are we using capitalize name?

Reply:
I was using an enum value directly. Changed it to small case.